.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,27 @@
+
+**What type of PR is this?**
+> Uncomment only one:
+>api-change
+>bug
+>cleanup
+>design
+>documentation
+>failing-test
+>feature
+
+**What this PR does / why we need it**:
```

Comment:
Also add a section to ask if the changes in the PR need to be cherry-picked to a release branch

Reply:
done